packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -96,4 +98,29 @@
                         Labels="@(ContainerLabel)"
                         ExposedPorts="@(ContainerPort)" />
     </Target>
+
+    <Target Name="PublishContainer" DependsOnTargets="$(PublishContainerDependsOn)"
+                                    Condition="'$(MSBuildRuntimeType)' == 'Full'">
+
+        <PropertyGroup Condition="'$(DOTNET_HOST_PATH)' == ''">
+            <DotNetHostDirectory>$(NetCoreRoot)</DotNetHostDirectory>
+            <DotNetHostFileName>dotnet</DotNetHostFileName>
+            <DotNetHostFileName Condition="'$(OS)' == 'Windows_NT'">dotnet.exe</DotNetHostFileName  >
+        </PropertyGroup>
+
+        <CreateNewImageToolTask ContainerizeDirectory="$(ContainerizeFolder)"
```

Comment:
Looks like this is missing `ExposedPorts`, which I think is just a symptom of my main concern about this PR which is: why do we need two separate task invocations?

Reply:
The new task also includes the DotNetRoot location parts, that might need some massaging to 'just work' inside the same Task invocation

Reply:
So are we thinking we only have `CreateNewImageToolTask` or (CreateNewImage becomes a tooltask itself) and we check inside the task itself if we're on Core/Full?

Reply:
Cleared this up. The ToolTask is being renamed to CreateNewImage